astronomer/airflow/version_check/update_checks.py

```diff
@@ -166,23 +150,14 @@ def check_for_update(self):
             return result, self.check_interval.total_seconds()
 
     def _process_update_json(self, update_document):
-        if 'runtime' in _app_name().lower():
-            return self._process_update_json_v1_0(update_document)
-        version = update_document.get('version', None)
-        if version != '1.0':
-            r = repr(version) if version else '<MISSING>'
-            raise RuntimeError("Un-parsable format_version " + r)
-
         return self._process_update_json_v1_0(update_document)
```

Comment:
Do we need this separate function still?

Reply:
Removed it